Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deltachat"
-version = "1.0.0-alpha.2"
+version = "1.0.0-alpha.3"
```

Comment:
why is everyone bumping versions in their PRs :'( ?

Reply:
because i want to know during testing that i am running against a new core ;)

Reply:
so alpha.3 should cure the ASM and import/export issue, if included in desktop releases